projects/nodejs/build.sh

```diff
@@ -19,28 +19,7 @@ cd $SRC/node
 # Build node
 export LDFLAGS="$CXXFLAGS"
 export LD="$CXX"
-./configure --without-intl --without-node-code-cache --without-dtrace --without-snapshot --without-ssl
+./configure --experimental-quic --with-ossfuzz
 make -j$(nproc)
 
-# Gather static libraries
-cd $SRC/node/out
-rm -rf ./library_files && mkdir library_files
-find . -name "*.a" -exec cp {} ./library_files/ \;
-
-# Build the fuzzers
-CMDS="-D__STDC_FORMAT_MACROS -D__POSIX__ -DNODE_HAVE_I18N_SUPPORT=1 \
- -DNODE_ARCH=\"x64\" -DNODE_PLATFORM=\"linux\" -DNODE_WANT_INTERNALS=1"
-INCLUDES="-I../src -I../deps/v8/include -I../deps/uv/include"
-
-# Compilation
-$CXX -o fuzz_url.o $SRC/fuzz_url.cc $CXXFLAGS $CMDS $INCLUDES \
-        -pthread -fno-omit-frame-pointer -fno-rtti -fno-exceptions -std=gnu++1y -MMD -c
-
-# Linking
-$CXX -o $OUT/fuzz_url $LIB_FUZZING_ENGINE $CXXFLAGS \
-  -rdynamic -Wl,-z,noexecstack,-z,relro,-z,now \
-  -pthread -Wl,--start-group \
-  ./Release/obj.target/cctest/src/node_snapshot_stub.o \
-  ./Release/obj.target/cctest/src/node_code_cache_stub.o \
-  fuzz_url.o ./library_files/*.a \
-  -latomic -lm -ldl -Wl,--end-group
+cp /src/node/out/Release/fuzz_url ${OUT}/
```

Comment:
I think just out/Release/fuzz_url should be better, we dont want to hardcode /src/node here.